pylint_odoo/checkers/format.py

```diff
@@ -17,6 +15,11 @@
         'no-utf8-coding-comment',
         settings.DESC_DFLT
     ),
+    'C%d02' % settings.BASE_FORMAT_ID: (
+        'UTF-8 coding is not necessary',
+        'unnecessary-utf8-coding-comment',
+        settings.DESC_DFLT
```

Comment:
Use original python version tuple format here too for the cases where a non-odoo-module is checked with this plugin